cdap-e2e-tests/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -3,3 +3,7 @@ clientUrl=http://localhost:11011
 serverUrl=https://placeholder.com/api
 # command to generate token: gcloud auth print-access-token
 serverAccessToken=placeholder
+bqSourceTable=dummy
```

Comment:
Add start and end comments for control center feature

Reply:
Added the comments